RELEASE.md

```diff
@@ -0,0 +1,54 @@
+eleasing a version
```

Comment:
Why this file?

Reply:
The file or the line? The file is because "a Google Doc somewhere that I can only find via browser history" is a bad location for maintainer docs.

The line is clipped-off copy paste accident, I'll fix that.